libs/libxx/__config_site

```diff
@@ -0,0 +1,69 @@
+/****************************************************************************
+ * libs/libxx/__config_site
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.  The
+ * ASF licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the
+ * License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ ****************************************************************************/
+
+/* Additional port configuration used by llvm libcxx.
+ * 
+ * This file is typically generated by CMake during configuration and used 
+ * to customize the build configuration for libcxx. As NuttX does not have 
+ * this functionality, this file is copied into the libcxx tree when NuttX 
+ * is initially built.
+ */
+
+
+#ifndef _LIBCPP_CONFIG_SITE
+#define _LIBCPP_CONFIG_SITE
```

Comment:
```suggestion
#ifndef __LIBS_LIBXX_LIBCPP___CONFIG_SITE
#define __LIBS_LIBXX_LIBCPP___CONFIG_SITE
```

Reply:
@pkarashchenko since this file is generated from a template(__config_site.in), it's better to avoid making the unnecessary modification.

Reply:
Sure. I didn't knew that is autogenerated